src/extension/php_objects.c

```diff
@@ -119,10 +157,79 @@ static zend_string *nullable _fetch_from_env(const char *name, size_t name_len)
     }
     *w = '\0';
 
+    return env_name;
+}
+
+/**
+ * This function is based on fpm_php_zend_ini_alter_master
+ * from php-src
+ */
+int _custom_php_zend_ini_alter_master(char *name, size_t name_length,
+    // NOLINTNEXTLINE(bugprone-easily-swappable-parameters)
+    char *new_value, size_t new_value_length, int mode, int stage,
+    bool has_env) /* {{{ */
```

Comment:
Nullability attributes (I'll stop mentioning this, but in general use them whenever possible).

Reply:
Done everywhere I think it should be used